tests/fmin.input

```diff
@@ -0,0 +1,66 @@
+# name fmin
+# arg1 decimal
+# arg2 decimal
+# ret  decimal
+
+0.0	0.0		0.0
+2.0	1.0		1.0
+1.0	2.0		1.0
+2.0	2.0		2.0
+-1.0	-2.0		-2.0
+-2.0	-1.0		-2.0
+-2.0	-2.0		-2.0
+2.123	1.86743		1.86743
+1.86744	1.86743		1.86743
```

Comment:
Many of those .input tests have extra spaces/tabs between the column. I am not sure about the code standards here but I guess we should not have those spaces here. But, I am not sure about it. Maybe @gftg85 or @tuliom  have a opinion about it.